app/src/main/java/cz/martykan/forecastie/models/Weather.java

```diff
@@ -72,6 +72,9 @@ public static int windDirectionDegreeToIndex(double degree, int numberOfDirectio
     private String lastUpdated;
     private Date sunrise;
     private Date sunset;
+    private double lat;
```

Comment:
According to Codacy, this whole block of field declarations needs to be at the top of the file. This was obviously already like this before but it's better to move the existing ones to the right place too.